project/BNFC.scala

```diff
@@ -92,7 +92,7 @@ object BNFC {
         // format: off
         javaSource     := (javaSource in Compile).value,
         scalaSource    := (javaSource in Compile).value,
-        bnfcNamespace  := "coop.rchain.rholang.syntax",
+        bnfcNamespace  := "coop.rchain.rholang.parser",
```

Comment:
@nzpr, @zsluedem I like _syntax_ more then _parser_ here, but we use syntax for extensions.
What do you think about `coop.rchain.rholang.ast`?

Reply:
ast is Ok to me

Reply:
parser is also ok to me.

Reply:
@vb-cs please change _parser_ to `ast`. It's more precise meaning for types generated from BNFC grammar.